google-translate-connector/processes/google/translate.p.json

```diff
@@ -0,0 +1,206 @@
+{
+  "format" : "10.0.0",
+  "id" : "1875AF047262A50C",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "google.translate.Data"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "getTranslatedText(String,String,String)",
+      "config" : {
+        "callSignature" : "getTranslatedText",
+        "input" : {
+          "params" : [
+            { "name" : "text", "type" : "String", "desc" : "The text to translate" },
+            { "name" : "targetLanguage", "type" : "String", "desc" : "The wished target language" },
+            { "name" : "sourceLanguate", "type" : "String", "desc" : "The source language" }
+          ],
+          "map" : {
+            "out.sourceLanguage" : "param.sourceLanguate",
+            "out.text" : "param.text"
+          },
+          "code" : [
+            "if (param.targetLanguage.isBlank()){",
+            "  param.targetLanguage = \"de\";",
+            "  ivy.log.info(\"parameter 'targetLanguage' undefined, assuming 'de'\");",
+            "}",
+            "",
+            "out.targetLanguage = param.targetLanguage;"
+          ]
+        },
+        "result" : {
+          "params" : [
+            { "name" : "translatedText", "type" : "String" }
+          ],
+          "map" : {
+            "result.translatedText" : "in.translatedText"
+          }
+        },
+        "tags" : "connector"
+      },
+      "visual" : {
+        "at" : { "x" : 208, "y" : 136 },
+        "labelOffset" : { "x" : -15, "y" : -23 },
+        "icon" : "res:/webContent/logo/google-translate-logo.png?small"
+      },
+      "connect" : { "id" : "f11", "to" : "f10" }
+    }, {
+      "id" : "f1",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 656, "y" : 136 },
+        "icon" : "res:/webContent/logo/google-translate-logo.png?small"
+      }
+    }, {
+      "id" : "f5",
+      "type" : "RestClientCall",
+      "name" : "Call Google Translator API",
+      "config" : {
+        "path" : "/language/translate/v2",
+        "clientId" : "3ee78482-7dbf-4c0e-9256-cb99472b4bcd",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "method" : "GET",
+        "queryParams" : {
+          "key" : "ivy.var.get(\"apiKey\")",
```

Comment:
maybe an obvious point, after adjusting the variables.yaml ... but this key should be qualified: `google-translate.apiKey` OR `google-translate-connector.apiKey` so that the administrator on the engine can have a clue what kind of apiKey that is.

Reply:
updated